app/models/plage_ouverture.rb

```diff
@@ -22,7 +22,7 @@ class PlageOuverture < ApplicationRecord
   has_many :webhook_endpoints, through: :organisation
 
   scope :for_lieu, ->(lieu) { where(lieu: lieu) }
-  scope :for_motif, ->(motif_name, organisation_id) { joins(:motifs).where(motifs: { name: motif_name, organisation_id: organisation_id }) }
+  scope :for_motif, ->(motif) { joins(:motifs).where(motifs: { id: motif.id }) }
```

Comment:
C’est pas `motif.plage_ouvertures` tout simplement? (On peut même retirer ce scope, je crois).

Reply:
:thinking: nous avons déjà `lieu.plage_ouvertures` je ne sais pas comment combiner les deux du coup

Reply:
`lieu.plage_ouvertures.merge(motif.plage_ouvertures)` ?

Reply:
It works. Je n'avais pas à l'esprit qu'il y avait une méthode `merge` dans ActiveRecord qui permet de maintenir une requête SQL... Merci.